Bonsai/Program.cs

```diff
@@ -41,6 +41,8 @@ static class Program
         [LoaderOptimization(LoaderOptimization.MultiDomainHost)]
         internal static int Main(string[] args)
         {
+            SystemResourcesExtensionsSupport.Initialize();
```

Comment:
Do we need to do this for every process, or just for the bootstrapper?

Reply:
I tried moving it to be bootstrapper-only and it goes bang trying to resolve `System.Runtime.CompilerServices.Unsafe` while loading the start screen. (The failed assembly load was initiated by `Bonsai.Configuration`, so it at least *tried*.)

As noted during our meeting I'm very uncomfortable with the idea of only doing this sometimes.

The dependencies of `System.Resources.Extensions` are all EOL and will not be updated so we will not need to handle the case of someone installing a package which wants a newer version.

`System.Resources.Extensions` itself still receives new versions, but I'm not very concerned about people wanting a different version from Bonsai. If someone does I will personally fly to their house and throw an plastic bonsai tree at them.

If I never have to think about ILRepacking this package again it will be too soon.